README.md

```diff
@@ -2,33 +2,16 @@
 
 ## Livongo-specific branches
 
-* `livongo-master-contrib`
-  * Master branch for Livongo's fixes/improvements to be contributed back to the upstream https://github.com/cakesolutions/scala-kafka-client
```

Comment:
this is probably still relevant

Reply:
I told Ivan that since the upstream project is dead, we don't need to bother with this branch any more.